app/views/protocol_importers/_protocol_card.html.erb

```diff
@@ -8,7 +8,10 @@
 
   <div class='row'>
     <div class='row col-md-12 info-line'>
-      <%= l(Time.at(protocol[:created_on]).to_datetime, format: :full_date) %> • <%= protocol[:authors] %>
+      <% if protocol[:published_on] %>
+        <%= l(Time.at(protocol[:published_on]).to_datetime, format: :full_date) %> •
+    <% end %>
```

Comment:
Indentation mismatch. Can .published_on fail for some reason?

Reply:
For now, we are getting time in UNIX timestamp so it shouldn't fail if `if` pass. Indentation updated.